crate/kms_client/src/kms_rest_client.rs

```diff
@@ -115,6 +109,18 @@ impl KmsClient {
         self.post_ttlv_2_1::<Create, CreateResponse>(&request).await
     }
 
+    pub async fn sign(&self, request: Sign) -> Result<SignResponse, KmsClientError> {
```

Comment:
Please add a header comment to this function. The implicit convention is to take the KMIP definition.